### PR TITLE
Introduce improvements to  the  Finalize Build Cache Configuration Build Operation

### PR DESCRIPTION
**Pull request by felipefzdz**
As a follow up of https://github.com/gradle/gradle/pull/1945

### CODE REVIEW

File: subprojects/core/src/integTest/groovy/org/gradle/caching/internal/FinalizeBuildCacheConfigurationBuildOperationIntegrationTest.groovy [hunk 1]
```diff
@@ -190,10 +198,80 @@ class FinalizeBuildCacheConfigurationBuildOperationIntegrationTest extends Abstr
 
         then:
         def result = result()
+        result.enabled == false
+
         result.local == null
```
Comment (alkemist) [inline subprojects/core/src/integTest/groovy/org/gradle/caching/internal/FinalizeBuildCacheConfigurationBuildOperationIntegrationTest.groovy]: @felipefzdz this doesn't seem right. Should we also be capturing the caches in this case?

Comment (felipefzdz) [inline subprojects/core/src/integTest/groovy/org/gradle/caching/internal/FinalizeBuildCacheConfigurationBuildOperationIntegrationTest.groovy]: True. I'll change it.

Comment (felipefzdz) [inline subprojects/core/src/integTest/groovy/org/gradle/caching/internal/FinalizeBuildCacheConfigurationBuildOperationIntegrationTest.groovy]: @alkemist a new commit has been pushed to address that.

**Comment by big-guy** [comment]
@alkemist @felipefzdz Thanks, this looks good.

**Pull request merged**